.github/workflows/wheels.yml

```diff
@@ -111,6 +112,15 @@ jobs:
               arch: i686
               build: "*manylinux*"
 
+          # additional manylinux variants, not specified in pyproject.toml:
+          # build with newer 2_28 for cpython >= 3.10, pypy 3.9
+          - name: manylinux-x86_64-2_28
+            os: ubuntu-20.04
+            cibw:
+              arch: x86_64
+              build: "cp31*-manylinux* pp39-manylinux*"
+              image: manylinux_2_28
```

Comment:
I think the `image` key is a too brief description since its configuring specifically a manylinux associated environment variable. I suggest re-using exactly the name of the env it will configure to reduce complexity of understanding whats going on.

Reply:
Good suggestion! Done.